ci-operator/step-registry/openshift/release/fips/openshift-release-fips-commands.sh

```diff
@@ -4,12 +4,51 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-if [[ "${RELEASE_IMAGE_LATEST}" == *"@sha256"* ]]; then
-    echo "digest based pullspecs is not supported"
-    exit 1
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+pass=true
+
+# get a master node
+master_node_0=$(oc get node -l node-role.kubernetes.io/master= --no-headers | grep -Ev "NotReady|SchedulingDisabled"| awk '{print $1}' | awk 'NR==1{print}')
+if [[ -z $master_node_0 ]]; then
+    echo "Error master node0 name is null!"
+    pass=false
+fi
+
+# create a ns
+namespace="fips-scan-payload-$RANDOM"
+run_command "oc create ns $namespace -o yaml | oc label -f - security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged --overwrite"
+
+if [ $? == 0 ]; then
+    echo "create $namespace namespace successfully"
+else
+    echo "Fail to create $namespace namespace."
+    pass=false
 fi
 
-# Get major.minor from payload url
-major_minor=$(echo "${RELEASE_IMAGE_LATEST}" | awk -F':' '{print $2}' | awk -F'.' '{print $1"."$2}')
 
-./check-payload scan payload -V "$major_minor" --url "${RELEASE_IMAGE_LATEST}"
+payload_pullspec=$(oc get clusterversion version -o json | jq -r .status.desired.image)
```

Comment:
I think this ends up just checking the payload image on the build cluster? You probably really do want to check `${RELEASE_IMAGE_LATEST}` so you're checking the payload under test

Reply:
Disregard, we cleared this up on Slack